examples/model_selection/plot_grid_search_stats.py

```diff
@@ -187,9 +187,10 @@ def corrected_std(differences, n_train, n_test):
 
     Returns
     -------
-    corrected_std : int
+    corrected_std : float
         Variance-corrected standard deviation of the set of differences.
     """
+    # kr = k times r, r times repeated k-fold crossvalidation
     kr = len(differences)
```

Comment:
Since in the end `kr` ends up being defined as `len(differences)` it might be worth adding as a comment here something like `kr equals the number of times the models were evaluated`

Reply:
@olszewskip Do you want to add that comment?

Reply:
Thank You for keeping me in the conversation. I took the liberty of modifying the suggested line a tiny bit.